[2023-06-10T01:22:45.627+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: fhv_taxi_data.format_to_parquet_task scheduled__2019-08-02T07:00:00+00:00 [queued]>
[2023-06-10T01:22:45.646+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: fhv_taxi_data.format_to_parquet_task scheduled__2019-08-02T07:00:00+00:00 [queued]>
[2023-06-10T01:22:45.647+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-06-10T01:22:45.648+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 2
[2023-06-10T01:22:45.649+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-06-10T01:22:45.663+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): format_to_parquet_task> on 2019-08-02 07:00:00+00:00
[2023-06-10T01:22:45.666+0000] {standard_task_runner.py:54} INFO - Started process 1562 to run task
[2023-06-10T01:22:45.686+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'fhv_taxi_data', 'format_to_parquet_task', 'scheduled__2019-08-02T07:00:00+00:00', '--job-id', '70', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_multiFile_v02.py', '--cfg-path', '/tmp/tmp7di5_8cf']
[2023-06-10T01:22:45.687+0000] {standard_task_runner.py:83} INFO - Job 70: Subtask format_to_parquet_task
[2023-06-10T01:22:45.691+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/***/dags/data_ingestion_gcs_multiFile_v02.py
[2023-06-10T01:22:45.804+0000] {data_ingestion_gcs_multiFile_v02.py:20} INFO - dtc_data_lake_dtc-dezc-001 is the bucket.
[2023-06-10T01:22:45.832+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:545: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  option = self._get_environment_variables(deprecated_key, deprecated_section, key, section)

[2023-06-10T01:22:45.899+0000] {task_command.py:384} INFO - Running <TaskInstance: fhv_taxi_data.format_to_parquet_task scheduled__2019-08-02T07:00:00+00:00 [running]> on host f41d61956f54
[2023-06-10T01:22:45.930+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:204: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2023-06-10T01:22:45.972+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=fhv_taxi_data
AIRFLOW_CTX_TASK_ID=format_to_parquet_task
AIRFLOW_CTX_EXECUTION_DATE=2019-08-02T07:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-08-02T07:00:00+00:00
[2023-06-10T01:22:46.075+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 193, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/data_ingestion_gcs_multiFile_v02.py", line 33, in format_to_parquet
    table = pv.read_csv(src_file)
  File "pyarrow/_csv.pyx", line 886, in pyarrow._csv.read_csv
  File "pyarrow/_csv.pyx", line 895, in pyarrow._csv.read_csv
  File "pyarrow/error.pxi", line 143, in pyarrow.lib.pyarrow_internal_check_status
  File "pyarrow/error.pxi", line 114, in pyarrow.lib.check_status
OSError: zlib inflate failed: incorrect header check
[2023-06-10T01:22:46.084+0000] {taskinstance.py:1406} INFO - Marking task as UP_FOR_RETRY. dag_id=fhv_taxi_data, task_id=format_to_parquet_task, execution_date=20190802T070000, start_date=20230610T012245, end_date=20230610T012246
[2023-06-10T01:22:46.095+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 70 for task format_to_parquet_task (zlib inflate failed: incorrect header check; 1562)
[2023-06-10T01:22:46.124+0000] {local_task_job.py:164} INFO - Task exited with return code 1
[2023-06-10T01:22:46.151+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-06-11T01:27:18.265+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: fhv_taxi_data.format_to_parquet_task scheduled__2019-08-02T07:00:00+00:00 [queued]>
[2023-06-11T01:27:18.273+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: fhv_taxi_data.format_to_parquet_task scheduled__2019-08-02T07:00:00+00:00 [queued]>
[2023-06-11T01:27:18.274+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-06-11T01:27:18.274+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 2
[2023-06-11T01:27:18.275+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-06-11T01:27:18.283+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): format_to_parquet_task> on 2019-08-02 07:00:00+00:00
[2023-06-11T01:27:18.287+0000] {standard_task_runner.py:54} INFO - Started process 6038 to run task
[2023-06-11T01:27:18.291+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'fhv_taxi_data', 'format_to_parquet_task', 'scheduled__2019-08-02T07:00:00+00:00', '--job-id', '613', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_multiFile_v02.py', '--cfg-path', '/tmp/tmp1a1wt7l6']
[2023-06-11T01:27:18.292+0000] {standard_task_runner.py:83} INFO - Job 613: Subtask format_to_parquet_task
[2023-06-11T01:27:18.293+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/***/dags/data_ingestion_gcs_multiFile_v02.py
[2023-06-11T01:27:18.359+0000] {data_ingestion_gcs_multiFile_v02.py:20} INFO - dtc_data_lake_dtc-dezc-001 is the bucket.
[2023-06-11T01:27:18.376+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:545: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  option = self._get_environment_variables(deprecated_key, deprecated_section, key, section)

[2023-06-11T01:27:18.418+0000] {task_command.py:384} INFO - Running <TaskInstance: fhv_taxi_data.format_to_parquet_task scheduled__2019-08-02T07:00:00+00:00 [running]> on host f41d61956f54
[2023-06-11T01:27:18.441+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:204: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2023-06-11T01:27:18.469+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=fhv_taxi_data
AIRFLOW_CTX_TASK_ID=format_to_parquet_task
AIRFLOW_CTX_EXECUTION_DATE=2019-08-02T07:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-08-02T07:00:00+00:00
[2023-06-11T01:27:19.073+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-06-11T01:27:19.083+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=fhv_taxi_data, task_id=format_to_parquet_task, execution_date=20190802T070000, start_date=20230611T012718, end_date=20230611T012719
[2023-06-11T01:27:19.197+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-06-11T01:27:19.241+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
